src/libraries/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -1040,14 +1040,21 @@ public sealed partial class JsonSourceGenerationOptionsAttribute : System.Text.J
         public System.Text.Json.Serialization.JsonKnownNamingPolicy PropertyNamingPolicy { get { throw null; } set { } }
         public bool WriteIndented { get { throw null; } set { } }
     }
-    [System.Diagnostics.CodeAnalysis.RequiresDynamicCodeAttribute("JsonStringEnumConverter cannot be statically analyzed and requires runtime code generation. Consider authoring a custom converter that is not a factory to work around the issue. See https://github.com/dotnet/runtime/issues/73124.")]
+    [System.Diagnostics.CodeAnalysis.RequiresDynamicCodeAttribute("JsonStringEnumConverter cannot be statically analyzed and requires runtime code generation. Native AOT applications should use the generic JsonStringEnumConverter<TEnum> instead.")]
```

Comment:
```suggestion
    [System.Diagnostics.CodeAnalysis.RequiresDynamicCodeAttribute("JsonStringEnumConverter cannot be statically analyzed and requires runtime code generation. Applications should use the generic JsonStringEnumConverter<TEnum> instead.")]
```

Ideally we want everybody to use the new one - it should be slightly faster in CoreCLR as well.